digitalocean/database/resource_database_cluster.go

```diff
@@ -532,6 +562,58 @@ func waitForDatabaseCluster(client *godo.Client, d *schema.ResourceData, status
 	return nil, fmt.Errorf("Timeout waiting to database cluster to become %s", status)
 }
 
+func waitForDatabaseBackups(client *godo.Client, d *schema.ResourceData, originalDatabaseName string) error {
+	var (
+		tickerInterval = 15 * time.Second
+		timeoutSeconds = d.Timeout(schema.TimeoutDelete).Seconds()
```

Comment:
Not sure this needs to be user configurable, but either way `TimeoutDelete` probably isn't what we want to use here. It returns 20m by default. 


https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2@v2.24.1/helper/schema#ResourceData.Timeout

Reply:
Oh yes I see- thanks for the docs!